spec/account-based-ticketing.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title: Account Based Ticketing
+hide_hero: true
+permalink: /spec/account-based-ticketing/
+---
+
+## Table of contents
+
+1. [Introduction](#introduction)
+2. [Use Cases](#usecases)
+2. [Process](#process)
+
+## Introduction <a name="introduction">
+
+The processes and the OSDM features to implement account based ticketing are described.
+
+OSDM upports the booking process where all or part of the offer and booking is made on an existing account of a provider. OSDM does not implement the creation of an account.
+
+Travel accounts in OSDM can be: 
+
+- TravelPassAccount
+
+   A travel account valid in a region. The travel account includes the region and the consumption.
+
+
+- MultiRideAccount
+
+  A travel account valid in a region but with a limited number of rides. The travel account includes the region, the balance on used rides and the consumption.
+
+- ReductionCardAccount
+
+  A reduction card used as travel account, e.g. cards providing free travel. The travel account contains the reduction card type.
+
+
+All travel Accounts include:
+
+- the issuer of the account
+- the person owning the account
+- the numer identifying the account
+
+
+The consumption on an account can be in money or any self-defined unit.
+
+
+## Use Cases <a name="usecases">
+
+
+### View the current consumption on a travel account
+
+```
+    GET /travel-accounts/{id}
+ ```
+returns the TravelAccountConsumption.
+
+### Getting offers on a travel account
+
+As a travel account is personal data it is not possible to request offers directly with a travel account.
```

Comment:
This needs further discussion or design because the original idea was to provide ticket number of a regional pass, system reads regional or other validity, and based on that provides offer parts to cover uncovered segments of trip.

Reply:
Yes, the documentation covers the current implementation only. Maybe we need an option to add an anonymous ticket and patch the number later.